config.yaml.tpl

```diff
@@ -1,14 +1,10 @@
-#!/bin/sh
 
-mkdir -p genconf
-cat << 'EOF' | sed '/^$/d' | tee genconf/config.yaml
----
 # Auto-generated by Terraform Templates
 # Created on date: ${timestamp()}
 ${bootstrap_private_ip == "" ? "" : "bootstrap_url: http://${bootstrap_private_ip}:${dcos_bootstrap_port}"}
 ${dcos_cluster_name == "" ? "" : "cluster_name: ${dcos_cluster_name}"}
 ${dcos_security== "" ? "" : "security: ${dcos_security}"}
-${dcos_resolvers== "" ? "" : "resolvers: ${dcos_resolvers}"}
+${dcos_resolvers== "" ? "" : "resolvers:  \n  ${indent(2,dcos_resolvers)}"}
```

Comment:
I expect this to be an error here. Does this work on your end when you tested?:
```
> list("1.2.3.4" , "4.5.6.7")
[
  "1.2.3.4",
  "4.5.6.7",
]
> indent(2,list("1.2.3.4" , "4.5.6.7"))
At column 3, line 1: indent: argument 2 should be typed string, got type list in:
```

The change made here to convert the dcos_resolvers from a string to a list should cause this to fail when it attempts to be evaluated.


```
> indent(2,"\n - 1.2.3.4\n - 4.5.6.7")

   - 1.2.3.4
   - 4.5.6.7
```